lib/configs/flat/addon-interactions.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * IMPORTANT!
+ * This file has been automatically generated,
+ * in order to update its content, execute "pnpm run update-all"
+ */
+export = [
+  {
+    name: 'storybook:addon-interactions:setup',
+    plugins: {
+      get storybook() {
+        return require('../../index')
+      },
+    },
+  },
+  {
+    name: 'storybook:addon-interactions:stories-rules',
+    files: ['*.stories.@(ts|tsx|js|jsx|mjs|cjs)', '*.story.@(ts|tsx|js|jsx|mjs|cjs)'],
```

Comment:
Shouldn't this be **/*.sto... ?

I tried out the canary build and it's not working for me. It's not searching recursively?

Reply:
@yannbf can you deploy a new canary with a fix for the glob pattern?

Reply:
There you go! `0.10.0--canary.156.408aed4.0`